net.jammr.jammr.json

```diff
@@ -0,0 +1,135 @@
+                }
+            ]
+        },
+        {
+            "name": "desktop",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 net.jammr.jammr.desktop /app/share/applications/net.jammr.jammr.desktop"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "net.jammr.jammr.desktop"
+                }
+            ]
+        },
+        {
+            "name": "portaudio",
+            "config-opts": [
+                "--disable-static",
+                "--without-oss"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.portaudio.com/archives/pa_stable_v190600_20161030.tgz",
+                    "sha256": "f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513"
+                }
+            ]
+        },
+        {
+            "name": "portmidi",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=/app/lib",
+                "-DCMAKE_ARCHIVE_OUTPUT_DIRECTORY=/app/lib",
+                "-DCMAKE_RUNTIME_OUTPUT_DIRECTORY=/app/bin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/project/portmedia/portmidi/217/portmidi-src-217.zip",
+                    "sha256": "08e9a892bd80bdb1115213fb72dc29a7bf2ff108b378180586aa65f3cfd42e0f"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/portmidi/portmidi-no-java.patch"
+                }
+            ],
+            "cleanup": [
+                "/bin"
+            ]
+        },
+        {
+            "name": "jammr",
+            "buildsystem": "qmake",
+            "config-opts": [
+                "QTCLIENT_TARGET=jammr",
+                "VERSION=1.2.8",
+                "APPNAME=jammr",
+                "ORGNAME=jammr",
+                "ORGDOMAIN=jammr.net",
+                "JAMMR_API_URL=https://jammr.net/api/",
+                "JAMMR_REGISTER_URL=https://jammr.net/accounts/register/",
+                "JAMMR_UPGRADE_URL=https://jammr.net/pricing.html",
+                "JAMMR_UPDATE_URL=https://jammr.net/static/latest-linux.txt",
+                "JAMMR_DOWNLOAD_URL=https://jammr.net/download.html"
+            ],
+            "make-install-args": [
+                "INSTALL_ROOT=/app"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/wahjam/wahjam.git",
+                    "tag": "jammr-1.2.8"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/wahjam/avoid-zero-length-vla.patch"
+                }
+            ]
+        }
+    ],
+    "cleanup": [
```

Comment:
Nitpick: the cleanup by convention goes just after the finish-args

Reply:
Thanks, will move in the next revision.